gcc/rust/ast/rust-ast-full-test.cc

```diff
@@ -4695,7 +4695,10 @@ MetaNameValueStr::to_attribute () const
 {
   LiteralExpr lit_expr (str, Literal::LitType::STRING,
 			PrimitiveCoreType::CORETYPE_UNKNOWN, {}, Location ());
-  return Attribute (SimplePath::from_str (ident),
+  // FIXME: What location do we put here? Is the literal above supposed to have
+  // an empty location as well?
+  // Should MetaNameValueStr keep a location?
+  return Attribute (SimplePath::from_str (ident, Location ()),
```

Comment:
Yeah this is because MetaNameValueStr is missing location info, there are quite a few places we have missing location info. Raise issues for each of these but don't fix them as part of this PR.

Just preserve this FIXME comment and over time raise issues for each of them, tagged as good-first-pr.